vault/resource_database_secret_backend_connection_test.go

```diff
@@ -703,6 +703,41 @@ func TestAccDatabaseSecretBackendConnection_snowflake(t *testing.T) {
 	})
 }
 
+func TestAccDatabaseSecretBackendConnection_redshift(t *testing.T) {
+	MaybeSkipDBTests(t, dbBackendRedshift)
+
+	url := os.Getenv("REDSHIFT_URL")
+	if url == "" {
+		t.Skip("REDSHIFT_URL not set")
+	}
+	backend := acctest.RandomWithPrefix("tf-test-db")
+	name := acctest.RandomWithPrefix("db")
+	resource.Test(t, resource.TestCase{
+		Providers:    testProviders,
+		PreCheck:     func() { testAccPreCheck(t) },
```

Comment:
`testAccPreCheck()` was just replaced by `testutil.TestAccPreCheck()` in 5660a057265b1eb2c00c7c2015de01979caa42f6, you'll want to merge/rebase.

Reply:
Fixed in 8185ea1